.github/workflows/acceptance-tests.yml

```diff
@@ -0,0 +1,112 @@
+name: Acceptance tests
+
+on:
+  pull_request:
+    types:
+    - opened
+    - edited
+    - synchronize
+    - reopened
+    - closed
+    paths:
+    - renku/service
+    - renku/core
```

Comment:
I believe this should be `renku/core/**`, otherwise it won't match on subfiles/folders. At least this was the case in the UI.
Reference: https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions#onpushpull_requestpaths